app/dashboard/settings/page.tsx

```diff
@@ -28,7 +37,25 @@ export default async function Settings() {
         />
       </ResponsiveContainer>
       <ResponsiveContainer className="gap-4">
-        <VersionSection installationIdPromise={installationIdPromise} />
+        <VersionSection />
+        <SettingsSection heading="Installation ID">
+          <Paragraph margin="none">
+            This is the unique identifier for your installation of Fresco. This
+            ID is used to track analytics data and for other internal purposes.
+          </Paragraph>
+
+          {!!env.INSTALLATION_ID && (
+            <Alert variant="info" className="mt-4">
+              <AlertTitle>Note:</AlertTitle>
+              <AlertDescription>
+                This setting is controlled by your <code>.env</code> file, and
+                so can only be changed by modifying that file.
+              </AlertDescription>
+            </Alert>
+          )}
+          <Input disabled={!!env.INSTALLATION_ID} value={installationId} />
```

Comment:
It should be possible to share the update functionality for this with the uploadthing editable input